README.md

```diff
@@ -197,9 +197,10 @@ class PagePreviewAPIViewSet(PagesAPIViewSet):
     )
 
     def listing_view(self, request):
-        page = self.get_object()
```

Comment:
This seems reasonable as it saves us maintaining duplicate code paths. I know this is just example code in a readme, but the question I'm left asking myself is why is the listing view the same as the detail view in the first place. Listing views provide a different subset of fields, why is that subset not supported for the preview. It seems semantically incorrect to return a detail view in place of a listing view. Why doesn't the listing return a 404 if it is not used.  @zerolab do you have any context for this? blame says you committed this chunk.

Reply:
This endpoint is for previewing a single page. This, if I recall correctly, is intended to shortcircuit non-detail uses

Reply:
It's so that you can call the endpoint without providing a (fake) PK (which is ignored anyway). wagtail-spa-integration for example uses this method.